pages/content/ch3/sets.mdx

```diff
@@ -0,0 +1,90 @@
+import { Callout, Tabs} from 'nextra/components'
+import Problem from '@/components/problem'
+
+# The most basic object
+
+In mathematics a set constitutes the most basic object. We define
+a mathematical object by the properties of the operations we can perform on it.
+
+In the case of sets, we only have a single operation: the membership predicate taking two
+sets and mapping them to a truth value. In infix notation:
+$$
+x \in A \\
+x \notin A \iff \neg (x \in A)
+$$
+
+Some sets can be written by enumerating its elements:
+$$
+\{ A, B, C \}
+$$
+
+<Callout type="warning" emoji="💡">
+  When enumerating sets, the order of elements does not matter. Elements
+  can be repeated without an effect:
+    $$ \{ A, B, C \} = \{ A, C, B \} = \{ A, B, C, A \} $$
+</Callout>
+
+In the following the define some shorthand notation and some properties
+of the membership predicate. If you want a more formal treatment of set theory
+have a look at the
+[Zermelo-Fraenkel set theory](https://en.wikipedia.org/wiki/Zermelo%E2%80%93Fraenkel_set_theory).
+
+<Callout type='axiom'>
```

Comment:
Das sind nicht die ZF axiome, oder? sind die aequivalent? wenn nicht, sind die sinnlos

Reply:
Ds ist einfach eine Zusammenfassung des Skripts

Reply:
Wuerde ich aber nicht Axiome nennen

Reply:
Steht auch nirgendwo

Reply:
achso, da steht aber "<Callout type='axiom'>" 😅/"Axiom of pairing". Sollte dann aber ok sein